### ISSUE TITLE
Is there plans to introduce distributed GPU training?

### ISSUE THREAD

**Issue by InfiniteLife** [google/automl#261]
Currently package is restricted to training models with very small batch size for large models. I managed to run D0 training with batch = 16 and D5 with batch = 1 on single GPU. I think this batch is too small to achieve introduced in paper accuracy gains. Partially this will be alleviated by GPU distributed training. But training larger models will be impossible because they simply do not fit in memory even with batch = 1. I have GTX 1080 Ti with 11 Gb. 
Where so much memory in tensorflow goes to? Is there a way to workaround this issue? 
Btw I tried creating optimizer with flag `aggregation_method=tf.AggregationMethod.EXPERIMENTAL_ACCUMULATE_N)` 
but it didnt helped. 

Edit: I was wrong, cant even run D5 with batch = 1

**Comment by mad-fogs** [comment]
This is the most top-0 issue.
